html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <header>
+    <h1>Coding Form!</h1>
+  </header>
+  <form>
+    <div>
+      Email
+      <br>
```

Comment:
make sure to remove any `br` tags from this HTML page, you want to use `div`s to force input onto another line